ner_v2/detectors/temporal/date/hi/date_detection.py

```diff
@@ -0,0 +1,67 @@
+from ner_v2.detectors.temporal.constant import LANGUAGE_DATA_DIRECTORY
+from ner_v2.detectors.temporal.date.standard_regex_date import BaseRegexDate
+from ner_v2.constant import TYPE_EXACT
+import os
+import re
+
+
+class DateDetector(BaseRegexDate):
+    def __init__(self, entity_name, timezone='UTC', past_date_referenced=False):
+
+        data_directory_path = os.path.join(os.path.dirname(os.path.abspath(__file__)).rstrip(os.sep),
+                                           LANGUAGE_DATA_DIRECTORY)
+        super(DateDetector, self).__init__(entity_name=entity_name, timezone=timezone,
+                                           data_directory_path=data_directory_path,
+                                           is_past_referenced=past_date_referenced)
+
+        self.custom_detectors = [self.custom_chritmast_date_detector]
+
+    def custom_chritmast_date_detector(self, date_list=None, original_list=None):
+        """
+        Method to detect chritmast
+        Args:
+
+        Returns:
+            date_list (list): list of dict containing day, month, year from detected text
+            original_list (list): list of original text corresponding to values detected
+
+        """
+        date_list = date_list or []
+        original_list = original_list or []
+
+        chritmas_regex = re.compile(r'((chritmas|xmas|x-mas|chistmas))')
```

Comment:
typo, same as above.
Also, why do we need to add a custom christmas date detector to Hindi one ?

Reply:
To give an example of how to add new pattern, Added same in readme also